docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+services:
+  mongo:
+    image: mongo:5
+    restart: always
```

Comment:
I would use `on-failure` here.

The `always` policy starts the service after restarting the docker daemon, even the service was stopped manually

Reply:
Changed.